File: tune_theta.m
function [ diff ] = tune_theta( theta,U,Pop)
global Fish Turf System

        out = GrowPopulation(Pop,U,'EQ',1,2,'No','f', theta);
        
        diff = (out.FinalMarginalProfits(1) - out.FinalMarginalProfits(2));

end

File: qTURF R Functions/tune_theta.R
tune_theta = function(theta,
                      total_effort,
                      scene,
                      patches,
                      time,
                      start_pop,
                      kmode = 'global',
                      stock_effect) {

  game_result = sim_pop(
    scene = scene,
    patches = patches,
    start_pop = start_pop,
    effort = total_effort * c(theta, 1 - theta),
    time = time,
    stock_effect = stock_effect,
    kmode = kmode
  ) %>%
    filter(year == max(year)) %>%
    select(marginal_profits)

  # return(diff(game_result$marginal_profits))

  return(diff(game_result$marginal_profits)^2)


}